{
  "uid" : "19363a71639662a7",
  "name" : "Ensuring the CA can issue certificates",
  "fullName" : "Auto-Generated Issuing CAs: Ensuring the CA can issue certificates",
  "historyId" : "a700d82052705bd37a5e530ace739162",
  "time" : {
    "start" : 1746708252540,
    "stop" : 1746708253113,
    "duration" : 573
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: STEP: When the admin attempts to issue a certificate using the generated CA\n",
  "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_008_steps.py\", line 126, in step_when_admin_issues_certificate\n    raise AssertionError(msg)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: STEP: When the admin attempts to issue a certificate using the generated CA\n",
    "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_008_steps.py\", line 126, in step_when_admin_issues_certificate\n    raise AssertionError(msg)\n",
    "steps" : [ {
      "name" : "Given the admin user is logged into TPC_Web",
      "time" : {
        "start" : 1746708252542,
        "stop" : 1746708253083,
        "duration" : 541
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When the admin attempts to issue a certificate using the generated CA",
      "time" : {
        "start" : 1746708253083,
        "stop" : 1746708253083,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: STEP: When the admin attempts to issue a certificate using the generated CA\n",
      "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_008_steps.py\", line 126, in step_when_admin_issues_certificate\n    raise AssertionError(msg)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Then the certificate issuance should succeed",
      "time" : {
        "start" : 1746708253113,
        "stop" : 1746708253113,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 3,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Auto-Generated Issuing CAs"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 3,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 3
      },
      "items" : [ {
        "uid" : "1633a025cbc3b6c8",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/17/#testresult/1633a025cbc3b6c8",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: When the admin attempts to issue a certificate using the generated CA\n",
        "time" : {
          "start" : 1746705364038,
          "stop" : 1746705364610,
          "duration" : 572
        }
      }, {
        "uid" : "f84b3571d7ede069",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/16/#testresult/f84b3571d7ede069",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: When the admin attempts to issue a certificate using the generated CA\n",
        "time" : {
          "start" : 1746616915368,
          "stop" : 1746616915985,
          "duration" : 617
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "19363a71639662a7.json",
  "parameterValues" : [ ]
}